src/Integrations/Greenhouse/GreenhouseClient.php

```diff
@@ -158,8 +176,6 @@ public function postApplication(string $itemId, array $params, array $files, str
 			];
 		}
 
-		$code = $response['response']['code'] ? $response['response']['code'] : 200;
-
 		if ($code === 200) {
```

Comment:
What if some other 200 code is returned? That's a successful code. Again, I'd check for ranges instead of a specific code.

Reply:
fixed